docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
```

Comment:
@CarmiWisemon 
"If you are a project architect, you..."
"...configure the instance's key parameters..."